packages/eslint-plugin-lit-a11y/lib/utils/isLitHtmlTemplate.js

```diff
@@ -4,37 +4,56 @@
  */
 
 /**
- * Whether a node is a lit-element css-tagged template expression
+ * Whether a node is a lit-html html-tagged template expression
  * @param {import('eslint').Rule.Node} node
- * @returns {node is LitTaggedExpression<'css'>}
+ * @param {import('./HasLitHtmlImportRuleExtension').LitA11yRuleContext} context
+ * @returns {node is LitTaggedExpression<'html'>}
  */
-function isCssTaggedTemplate(node) {
-  return (
-    node.type === 'TaggedTemplateExpression' &&
-    node.tag.type === 'Identifier' &&
-    node.tag.name === 'css'
-  );
+function isHtmlTaggedTemplate(node, context) {
+  if (node.type !== 'TaggedTemplateExpression') return false;
+  const { litHtmlTags = ['html'], litHtmlNamespaces = [] } = context.parserServices || {};
+
+  switch (node.tag.type) {
+    case 'Identifier':
+      return litHtmlTags.includes(node.tag.name);
+    case 'MemberExpression':
+      return (
+        node.tag.property.type === 'Identifier' &&
+        node.tag.property.name === 'html' &&
+        node.tag.object.type === 'Identifier' &&
+        litHtmlNamespaces.includes(node.tag.object.name)
+      );
+    default:
+      return false;
+  }
 }
 
 /**
- * Whether a node is a lit-html html-tagged template expression
+ * Whether a node is a lit-element css-tagged template expression
  * @param {import('eslint').Rule.Node} node
- * @returns {node is LitTaggedExpression<'html'>}
+ * @param {import('./HasLitHtmlImportRuleExtension').LitA11yRuleContext} context
+ * @returns {node is LitTaggedExpression<'css'>}
  */
-function isHtmlTaggedTemplate(node) {
+// future iterations might want to apply CSS a11y checks,
+// at which point, imports should be checked against context just like for html
+// see also `isSvgTaggedTemplate`
+// eslint-disable-next-line no-unused-vars
+function isCssTaggedTemplate(node, context) {
   return (
     node.type === 'TaggedTemplateExpression' &&
     node.tag.type === 'Identifier' &&
-    node.tag.name === 'html'
+    node.tag.name === 'css'
   );
 }
 
 /**
  * Whether a node is a lit-html svg-tagged template expression
  * @param {import('eslint').Rule.Node} node
+ * @param {import('./HasLitHtmlImportRuleExtension').LitA11yRuleContext} context
  * @returns {node is LitTaggedExpression<'svg'>}
  */
-function isSvgTaggedTemplate(node) {
+// eslint-disable-next-line no-unused-vars
+function isSvgTaggedTemplate(node, context) {
```

Comment:
note for the future: we could add a rule to enforce accessible inline svgs https://css-tricks.com/accessible-svgs/